src/components/JobWizard/JobWizardSteps/JobWizardParameters/JobWizardParameters.js

```diff
@@ -69,6 +71,7 @@ const JobWizardParameters = ({ formState, stepIsActive }) => {
         exitEditModeTriggerItem={stepIsActive}
         fieldsPath={`${PARAMETERS_STEP}.parametersTable`}
         formState={formState}
+        id={hyperParametersAreEnabled ? 'hyper-parameters-form-table' : 'parameters-form-table'}
```

Comment:
The same as DataInputs table
https://github.com/mlrun/ui/pull/2105/files#diff-d3cad1895d38a31464eca651332455014e5c9bd08d99197c7b0b6fb55944897dR38

Reply:
I prefer this will have dynamic id based on the hyper. The table still has default id inside